site/docs/v1/tech/core-concepts/tenants.adoc

```diff
@@ -53,6 +54,21 @@ The named issuer used to sign tokens.  Typically a fully-qualified domain name.
 [field]#Login Theme# [optional]#Optional#::
 The Theme associated with this Tenant; determines which templates to render for interactive work-flows.
 
+===== Form Settings
+
+[.api]
+[field]#Admin user form# [optional]#Optional#::
+The Admin form used by FusionAuth for adding or editing users. FusionAuth provides a default form that will be used unless otherwise specified.
++
+Updating this fields requires a paid edition of FusionAuth.
+
+==== Connectors
+
+Connectors can be enabled on a per tenant basis with a Connector policy.
+
+image::connectors/tenant-connector-tab.png[The Tenant Connector policy configuration tab.,width=1200,role=shadowed]
```

Comment:
We should probably document the add connector policy dialog here. External JWT IdP has an example of a dialog.